extension/src/fileSystem/index.ts

```diff
@@ -136,6 +136,11 @@ export const scriptCommand = {
   PYTHON: 'python'
 }
 
+export const openDvcYamlFile = async (dvcYamlPath: string) => {
```

Comment:
[C] This name can be generalised (I think we open a file the same way in at least one test). Can replace the other usage